### PR TITLE
repose 7.x changes

### PR DESCRIPTION
**Pull request by mwhahaha**
Based on the changes in 7.0.0.1, the filter needs package updates, an annotation and rpm dependency updates.

https://repose.atlassian.net/wiki/display/REPOSE/Repose+Release+Notes

### CODE REVIEW

**Comment by shintasmith** [comment]
ok to test

**Comment by VinnyQ** [comment]
lgtm.

Do we also need to do this for the other filters in cloudfeeds-atomhopper and cloudfeeds-catalog?

**Comment by usnavi** [comment]
@VinnyQ - we need to run the system regression tests on this fix before we accept the PR.  This is deployed on @mwhahaha 's inova, right?  can you run them against that?

We only need to do this for repose-filters, and we only have one.

**Comment by VinnyQ** [comment]
@mwhahaha I don't see a .properties file for your nodes in atom-hopper-system-regression.  Alex can I have the ip address to the nodes you have this deployed on?

File: feeds-filters/pom.xml [hunk 1]
```diff
@@ -61,7 +61,7 @@
                             </mapping>
                         </mappings>
                         <requires>
-                            <require>repose</require>
```
Comment (mwhahaha) [inline feeds-filters/pom.xml]: In talking with the repose team, they might be putting the repose back into the RPMs so this might not be needed.

**Comment by usnavi** [comment]
@mwhahaha - I assume the current deployment on your repose 7 inova has this deployed on it?

**Comment by mwhahaha** [comment]
@usnavi yes

**Pull request merged**